### PR TITLE
[DJFS-111] Develop a Customers details list view page

### PR DESCRIPTION
**Pull request by ChabuChabu**
## Task: `List Customers in database
- `Save customers to cookie
- List customers

### CODE REVIEW

File: submissions/chabuChabu/phpDatabase/viewCustomers.php [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+            echo"<th>Full Name</th>";
+            echo"<th>Email Address</th>";
+            echo"<th>Created At</th>";
+            echo"<th>Action</th>";
+       echo"</tr>";
+       
+       foreach($result as $row){
+        echo"<tr>";
+           echo"<td>{$row['id']}</td>";
+           echo"<td>{$row['full_name']}</td>";
```
Comment (adaugochi) [inline submissions/chabuChabu/phpDatabase/viewCustomers.php:37]: there is no column on the `customsers` table with the name `full_name`

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/phpDatabase/viewCustomers.php (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/phpDatabase/viewCustomers.php:37]: @adaugochi I don't have a column in customers table called full_nane,what  have is first-name and last-name.But what i understand is that the task requires me to display these as Full Name on the browser and as a result am concatenating first_name and last_name..is my understanding wrong?

Comment (adaugochi) [reply submissions/chabuChabu/phpDatabase/viewCustomers.php:37]: I can't see where you are doing the concatenation

**Review by adaugochi — CHANGES_REQUESTED** [review]

File: submissions/chabuChabu/phpDatabase/viewCustomers.php (hunk 1 above)
Comment (ChabuChabu) [reply submissions/chabuChabu/phpDatabase/viewCustomers.php:37]: @adaugochi i have done it in the select query

**Pull request merged**